### DISCUSSION TITLE
Etherscan verification issue in Foundry-Fund-Me

### DISCUSSION THREAD

**Discussion by Muhdhammad** [Cyfrin/foundry-full-course-cu#2180] [General]
Hey everyone, so I have deployed my fundMe on Sepolia TestNet and it has been deployed successfully, here is the transaction hash: 0xdbe454ce56b1d5b83cea9abc99e308af961c01aab67db485962918a3cffa9c03
But, it is not getting verified this is the error I have received:
**Error:** 
Not all (0 / 1) contracts were verified!

**Comment by EngrPips** [comment]
Hello @Muhdhammad, You can follow the directions on the `Foundry Documentation` https://book.getfoundry.sh/forge/deploying to deploy and verify your contract properly.
